tests/certification/bindings/azure/blobstorage/blobstorage_test.go

```diff
@@ -0,0 +1,731 @@
+/*
+Copyright 2021 The Dapr Authors
```

Comment:
nit: 2022 on all headers.

Reply:
I am confused about the year here, need to update all the time?